featuretools/tests/primitive_tests/test_distancetoholiday_primitives.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from featuretools.primitives.standard.transform_primitive import (
+    DistanceToHoliday
+)
+
+
+def test_distanceholiday():
+    distance_to_holiday = DistanceToHoliday("New Year's Day")
+    dates = [datetime(2010, 1, 1),
+             datetime(2012, 5, 31),
+             datetime(2017, 7, 31),
+             datetime(2020, 12, 31)]
+
+    distance_list = distance_to_holiday(dates).tolist()
+
+    assert distance_list[0] == 0
```

Comment:
Minor suggestion, but can we create a list of expected values and assert the same way we do in `test_nat` and `test_valid_country` instead of checking each value individually?